matchbox_socket/src/webrtc_socket/mod.rs

```diff
@@ -165,6 +201,9 @@ impl WebRtcSocket {
         debug!("waiting for peers to join");
         let mut addrs = vec![];
         while let Some(id) = self.new_connected_peers.next().await {
+            if addrs.contains(&id) {
+                continue;
```

Comment:
Do we actually expect this to happen?

Reply:
This happened in "native" because a message was sent on the new_peer channel everytime a data channel was ready. This became an issue now because there can be several data channels per peer. However I fixed that now in the "native" code and will remove it here